app/Http/Controllers/GitHubSocialiteController.php

```diff
@@ -63,27 +63,26 @@ public function callback()
 
     protected function retrieveUser($gitHubUser): User
     {
-        if (session('auth-user-email')) {
+        if (session('auth-user-id')) {
```

Comment:
@riasvdv Kunnen we hier niet effectief checken of de user is ingelogd met `auth()->user()`? Ipv het in dezelfde session te moeten steken waar de logged in user ook in zit :p

Reply:
Denk dat dat niet altijd werkte en er soms een nieuwe sessie werd gestart om een of andere reden

Reply:
Nu ik er nog eens over nadenk, dan zou de sessie ook verloren zijn wanneer iemand uitgelogd wordt, denk dat we inderdaad wel met `auth()->user()` kunnen werken